### PR TITLE
add support for binom in latex

### PR DESCRIPTION
**Pull request by iammosespaulr**
#### Brief description of what is fixed or changed
Added support for `binomial`in latex and also added tests for it

#### Release Notes

* parsing
  * added support for `binom`

### CODE REVIEW

**Comment by iammosespaulr** [comment]
If the tests pass, I'll add support for a lot more combinatorial functions

**Comment by iammosespaulr** [comment]
@smichr wdyt ?

File: sympy/parsing/latex/_antlr/latexlexer.py [hunk 1]
```diff
@@ -331,15 +349,16 @@ class LaTeXLexer(Lexer):
...
+                  u"UNDERSCORE", u"CARET", u"COLON", u"WS_CHAR", u"DIFFERENTIAL",
+                  u"LETTER", u"DIGIT", u"NUMBER", u"EQUAL", u"LT", u"LTE",
+                  u"GT", u"GTE", u"BANG", u"SYMBOL" ]
 
     grammarFileName = u"LaTeX.g4"
 
     def __init__(self, input=None, output=sys.stdout):
         super(LaTeXLexer, self).__init__(input, output=output)
-        self.checkVersion("4.7.1")
+        self.checkVersion("4.7.2")
```
Comment (sylee957) [inline sympy/parsing/latex/_antlr/latexlexer.py:361]: I see this always gets modified across different versions.
I have rebuilt with 4.7.1 and I see some diffs.

Comment (iammosespaulr) [reply sympy/parsing/latex/_antlr/latexlexer.py:361]: The latest version `4.8` came out, do you want me to upgrade to that, seems to work fine locally.
or should I stick with `4.7.2` in which case I'll update `$ conda install -c conda-forge antlr=4.7` in `README.md`

Comment (iammosespaulr) [reply sympy/parsing/latex/_antlr/latexlexer.py]: I downgraded to 4.7.1 and yeah there were some differences. 😄

**Pull request merged**